### ISSUE TITLE
Files being left as read-only on disk

### ISSUE THREAD

**Issue by Acren** [ProjectBorealis/UE4GitPlugin#45]
Hey there and thanks for the plugin!

My team's been using it for a bit and there seems to be a somewhat common issue among us where files seem to be getting left as read-only even when they are checked out, and we're not sure what's causing it.

Then when Unreal tries to save the checked-out asset it fails because it's read-only, and as a result we have to spend a lot of time manually going into the filesystem in explorer and making files writable again so that we can save them.

![image](https://user-images.githubusercontent.com/1582409/174025291-df6a32af-eea9-4f53-9bab-4b6d97cf1e2c.png)

Has anyone encountered an issue like this before? We're using UE 5.0 release with 3.0.4 of this plugin.

Much thanks!

**Comment by stilobique** [comment]
This error can typically comming if you have 2 unreal editor open in your system (look your  task manager, sometime unreal can be start more than one).
Another way, the source control setup are correctly setup ? In the right ui, have you a green circle to comfirm everything are setup about the source control ?

**Comment by mastercoms** [comment]
do you have git hooks set up for LFS?

**Comment by SamBonifacio** [comment]
> Another way, the source control setup are correctly setup ? In the right ui, have you a green circle to comfirm everything are setup about the source control ?

Yep, always green circle and everything mostly works in editor.

> do you have git hooks set up for LFS?

I'm not too sure - LFS appears to be working normally in most cases. Is there a way we can check this? There seem to be LFS related hooks (post-commit, post-merge, pre-push etc) in the hooks directory, if that's any indication.

Thanks again

**Comment by Acren** [comment]
> Another way, the source control setup are correctly setup ? In the right ui, have you a green circle to comfirm everything are setup about the source control ?

Yep, always green circle and everything mostly works in editor.

> do you have git hooks set up for LFS?

I'm not too sure - LFS appears to be working normally in most cases. Is there a way we can check this? There seem to be LFS related hooks (post-commit, post-merge, pre-push etc) in the hooks directory, if that's any indication.

Thanks again

**Comment by kromond** [comment]
I'm seeing this also

**Comment by mastercoms** [comment]
Could you give more information about reproduction?

**Comment by kromond** [comment]
This has not happened again.  Sorry for the noise!

**Comment by SilentGodot** [comment]
Hi,
This is happening to me on the regular. Here are the steps to reproduce (on release 3.16):
1. Add a new LFS asset. Uasset file for example.
2. Save it. Marked for add etc.
3. Close unreal without commiting the change
4. Reopen unreal.
5. The file is now read only, and if you try to save it you will see a popup like the in the OP.

**Comment by sanxfxteam** [comment]
Hi,

I'm also experiencing this 100%. I'm also unable to do anything much really.
I followed the installation instructions (except I installed the plugin in the project not the editor) + I turned on lfs lock using the command line indicated in the logs.

Here is what I see if I right click in the Content browser:
![image](https://github.com/user-attachments/assets/a528091e-8d07-401e-980c-806064e6e676)
I don't have the option to checkout the file.

When I open the file and then try to save:
<img width="413" alt="image" src="https://github.com/user-attachments/assets/5e17c477-1bbd-4e52-b54b-be5e45aaa19a">

The only way to be able to edit a file is to manually lock the file
```
git lfs lock Project/Content/Blueprints/Actor/ActorName.uasset
```

Now I can edit, but when I commit, the file is still locked...

UE 5.3.2
Plugin 3.16
Azure Dev Ops Git Hosting
git 2.45.2.windows.1
git credentials

When I start the project although the revision control is enabled, I see this the Revision Control Log:
```
Revision control is disabled
Revision control is disabled
Revision control is disabled
Revision control is disabled
```
When I press Accept settings, no change happens.

Thanks for your help.
